src/app/app-routing.module.ts

```diff
@@ -142,6 +143,11 @@ export const ROUTES: DrawerItem[] = [
                 path: 'action-list',
                 component: ActionListComponent,
             },
+            {
```

Comment:
Added new route for global action list with panel header. (Not matching with react). 

Angular design
<img width="295" alt="Screenshot 2021-11-02 at 12 22 27 PM" src="https://user-images.githubusercontent.com/10433274/139799509-41b9ac16-c876-45b4-9cc0-68faa3a28b18.png">

React Design:
<img width="300" alt="Screenshot 2021-11-02 at 12 23 23 PM" src="https://user-images.githubusercontent.com/10433274/139799605-b0a79cec-5d21-4178-96fd-85b15d27f9ba.png">

Reply:
we called this out in react. There are plans to rename parent dir for both